src/lib/ElectrumClient.js

```diff
@@ -467,7 +467,7 @@ export default class Client {
  * @param {string} script ScriptPubKey in a hexadecimal format.
  * @return {string} Script hash as a hex string.
  */
-function scriptToHash(script) {
+export function scriptToHash(script) {
```

Comment:
Wish there was a way to export this for testing without exporting it for everyone else 🤔

Reply:
I thought of using something like `rewire` but given that this is an internal file which is not meant to be used as part of the library I thought it would be okay to just export it.

What do you think would be the best solution among these?
- Move it into it's own file and test that directly
- Spy on the implementation using `rewire`

Reply:
Sorry, I should have been clearer---I'm fine with it as is, don't think we need to do any additional work right now. I've been thinking about it some more and really, it might even make more sense as a function of `ElectrumClient`, which is where it was in the first place. Either way, I don't think it needs any more work right now, we are past Good Enough™.

Reply:
Got it! :ok_hand: